config/initializers/dinkurs.rb

```diff
@@ -0,0 +1,9 @@
+# Required to communicate with Dinkurs.se via the DinkursService object
+#
+# This gets information from ENV either the .env file or the actual environment via the dotenv gem.
+
+require 'dotenv'
+
+DINKURS_XML_URL =   ENV['DINKURS_XML_URL']
+
+DINKURS_COMPANY_ARG = ENV['DINKURS_URL_COMPANY_ARG']
```

Comment:
Is this needed anymore?  We have the Dinkurs company key now as an attribute to `Company`.

Reply:
It's not actually the company key (the attribute for `Company`).  It's the parameter name that DinKurs.se expects in the URL.  

Ex:  This is the URL that DinKurs.se expects for fetching data for company with key =`ABC123` : 
```
DINKURS_XML_URL + '?' + DINKURS_COMPANY_ARG + "=#{args[0][0]}"
=
 https://dinkurs.se/lxml.asp?company_key=ABC123
```

I surely overdid it with making DINKURS_COMPANY_ARG an ENV variable.   We could probably safely assume that "?company_key" is not going to change a lot and hardcode it.

Just my typical "overengineering" approach. ;-)